### ISSUE TITLE
[backlog] Update script like install script

### ISSUE THREAD

**Issue by anthony-sarkis** [diffgram/diffgram#258]
Reflecting that updating is going to be a regular occurrence.
It would be good to have a single script to run that handles the process.

Currently we have https://diffgram.readme.io/docs/open-core-docker-install-cheatsheet#update-to-latest-version
Which a user can chain/copy and paste as one command but I think we can can make it much better

**Comment by PJEstrada** [comment]
Yes 100% agree I think we should eventually have the diffgram-admin command to centralize all of these utilities. +1 for this one
